src/unix/lwt_unix.cppo.mli

```diff
@@ -409,6 +409,11 @@ sig
   (** [is_empty vs] is [true] if and only if [vs] has no I/O vectors, or all I/O
       vectors in [vs] have zero bytes. *)
 
+  val length : t -> int
```

Comment:
I think it's better to rename this, so it's completely clear from the name that it is not the number of vectors, but the number of bytes in the vectors. Something like `byte_count` may work.